usb/esp_tinyusb/idf_component.yml

```diff
@@ -1,7 +1,7 @@
 ## IDF Component Manager Manifest File
 description: Espressif's additions to TinyUSB
 documentation: "https://docs.espressif.com/projects/esp-idf/en/latest/esp32s2/api-reference/peripherals/usb_device.html"
-version: "1.4.2~2"
+version: "1.4.3"
```

Comment:
Please update the https://github.com/espressif/idf-extra-components/blob/master/usb/esp_tinyusb/CHANGELOG.md too